Documentation/ApiOverview/Fal/Architecture/Database.rst

```diff
@@ -83,6 +83,39 @@ The :sql:`sys_file_metadata` table is extended by the system extension
 `filemetadata`_. In particular, it adds the necessary definitions
 to categorize files with :ref:`system categories <categories>`.
 
+Also some other helpful metadata attributes are filled, most of them
+are self-explanatory; this list may not reflect the most recent TYPO3
+version, so it is recommended to inspect the actual TCA configuration
+of that table:
+
+*  :sql:`visible`
```

Comment:
what did you order the fields by?

Reply:
By their TCA order of definition

Reply:
which does have no semantical meaning? I think we should either order them in the order they are typically displayed in the backend or alphabetically..

Reply:
No problemo! Ordered alphabetically, I was not in the mood to cross-reference it with the palette :D